package.json

```diff
@@ -65,9 +68,11 @@
         "gh-pages": "^6.1.1",
         "postcss": "^8.4.38",
         "tailwindcss": "^3.4.4",
-        "vite": "^5.3.1"
+        "typescript": "^5.5.4",
+        "vite": "^5.3.1",
+        "vite-plugin-dts": "^4.0.0-beta.2"
     },
     "peerDependencies": {
         "@tanstack/react-query": "^5.51.15"
```

Comment:
Should `ol` and `plotly` be in here as well? A table library if we use that?

Reply:
Are you referring to their @types packages?  If so, I guess they can just be added as people need them.  I haven't used that stuff within groundwork-water so haven't had a need for them yet -- easy to add though if need be.